docs/book/getting_started/introduction.md

```diff
@@ -0,0 +1,75 @@
+---
+description: Start here with the ZenML Docs.
+---
+
+# Introduction
+
+Are you an ML engineer or data scientist shipping models to production and jumbling a plethora of tools?
+Do you struggle with versioning data, code, and models in your projects?
+Have you had trouble replicating production pipelines and monitoring models in production?
+
+If you answered yes to any, **ZenML** is here to help with all that, and more. 
+![Zenml](../assets/oss-header.svg)
+
+**ZenML** is an extensible, open-source MLOps framework for creating portable, production-ready **MLOps pipelines**. 
+It's built for data scientists, ML Engineers, and MLOps Developers to collaborate as they develop to production.
+**ZenML** has simple, flexible syntax, is **cloud-** and 
+**tool-agnostic**, and has interfaces/abstractions that are catered towards ML workflows.
+
+
+<!-- At its core, ZenML pipelines execute ML-specific workflows from sourcing
+data to splitting, preprocessing, training, all the way to serving and monitoring 
+ML models in production. There are many built-in features to support
+common ML development tasks. ZenML is not here to replace the great tools that
+solve these individual problems. Rather, it offers an **extensible framework** and a standard abstraction to write and build your workflows. -->
+
+## Why ZenML
```

Comment:
I feel like this section is a bit outdated and can be written with a more precise focus. I think people are looking for the orbital reason why ZenML needs to exist. Listing features might not be the best idea. I would rather say something like "If you come from unstructured notebooks or scripts, or have a manual process, ZenML will make the path to production easier and faster for you". We can also highlight integrations and portability here

Reply:
I feel like putting some contents+graphic from this section of the webpage in. But would it be a repetition if we do so?

![image](https://user-images.githubusercontent.com/6821286/179167225-ca67ccbf-c655-4cd4-baa6-6e099b0adb09.png)

Reply:
I've rewritten this section to view ZenML from a higher level perspective.